functions/lib/github_api.js

```diff
@@ -0,0 +1,81 @@
+var rp = require('request-promise')
+const functions = require('firebase-functions')
+
+module.exports = GitHubAPI
+/**
+ * Crowd-related functions.
+ * @param accessToken {string} Access Token for GitHub API
+ * @return {{getUsers: getUsers, addUser:addUser, deleteUser:deleteUser}}
+ * @constructor
+ */
+function GitHubAPI(accessToken) {
```

Comment:
We already use Octokit in github.js to interact with Github APIs - can you use that as well instead of defining http calls here?

Reply:
I use request-promise (rp) for crowd because I couldn't find a good library for that

Reply:
ok, I will try it